forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -543,6 +465,105 @@ const Edit = React.memo(() => {
   const formChange = (newForm) =>
     dispatchFormAction({ type: "formChange", value: newForm });
 
+  const handlePublish = () => {
+    if (!promptNewVersion) {
+      setIsPublishLoading(true);
+      if (processListData.status === "active") {
+        unPublish(processListData.id)
+          .then(() => {
+            setPromptNewVersion(true);
+            setIsPublishLoading(false);
+            setIsPublished(!isPublished);
+            setVersionSaved(true);
+          })
+          .catch((err) => {
+            setIsPublishLoading(false);
+            const error = err.response?.data || err.message;
+            dispatch(setFormFailureErrorData("form", error));
```

Comment:
Please use a reusable method in place of 480-482 & 493-495

Reply:
sure